src/webgl/loading.js

```diff
@@ -9,6 +9,204 @@
 import p5 from '../core/main';
 import './p5.Geometry';
 
+/**
+ * Load a 3d model from an OBJ or STL string.
+ *
+ * <a href="#/p5/loadModel">createModel()</a> should be placed inside of <a href="#/p5/preload">preload()</a>.
+ * This allows the model to load fully before the rest of your code is run.
+ *
+ * One of the limitations of the OBJ and STL format is that it doesn't have a built-in
+ * sense of scale. This means that models exported from different programs might
+ * be very different sizes. If your model isn't displaying, try calling
+ * <a href="#/p5/loadModel">loadModel()</a> with the normalized parameter set to true. This will resize the
+ * model to a scale appropriate for p5. You can also make additional changes to
+ * the final size of your model with the <a href="#/p5/scale">scale()</a> function.
+ *
+ * Also, the support for colored STL files is not present. STL files with color will be
+ * rendered without color properties.
+ *
+ * Options can include:
+ * - `path`: Specifies the plain text string of either an stl or obj file to be loaded.
```

Comment:
Rather than specifying a `path`, maybe we can make the `fileType` mandatory and place it after the source string for all the different overloads for this method?